cmd/virt-launcher/virt-launcher.go

```diff
@@ -329,6 +330,13 @@ func main() {
 
 	log.InitializeLogging("virt-launcher")
 
+	// check if virt-launcher verbosity should be changed
+	if verbosityStr, ok := os.LookupEnv("VIRT_LAUNCHER_LOG_VERBOSITY"); ok {
+		verbosity, _ := strconv.Atoi(verbosityStr)
+		log.Log.SetVerbosityLevel(verbosity)
+		log.Log.V(4).Infof("logs verbosity set to debug")
```

Comment:
should this log message contain the verbosity level like the others?

Reply:
oh that's right

Reply:
done